learn/src/main/java/org/navgurukul/learn/ui/learn/adapter/ExerciseSlugAdapter.kt

```diff
@@ -97,10 +99,10 @@ class ExerciseSlugAdapter(callback: (Exercise.ExerciseSlugDetail) -> Unit) :
         binding: ItemSlugDetailBinding
     ) {
         binding.youtubeView.visibility = View.GONE
-        binding.defaultTextView.visibility = View.GONE
+        binding.imageView.visibility = View.GONE
         binding.markDownContent.visibility = View.VISIBLE
         val value = item.value
-        var content = StringBuilder()
+        val content = StringBuilder()
         val gson = Gson()
         try {
```

Comment:
extension could help here as well, this repetition of gson code and try catch would be removed from here